brotli/FeatureModel.xml

```diff
@@ -1,3 +1,4 @@
+<!--Locations are based on commit 9801a2c5d6c67c467ffad676ac301379bb877fc3-->
```

Comment:
Thanks for adding this, it's important to note down the commits.
Did you check that all locations match in this commits?

Reply:
Yeah, I checked the locations.